app/services/booking_responder.rb

```diff
@@ -1,16 +1,20 @@
 class BookingResponder
   delegate :visit, to: :staff_response
 
-  def initialize(staff_response, message = nil)
-    @staff_response = staff_response
-    @message = message
+  def initialize(visit:, user: nil, message: nil)
```

Comment:
what about this then?
```ruby
BookingResponder.new(visit, user: user, message: message)
```
To avoid passing `nil` for the `user:` if you want to pass a message